docs/blog/2021-12-09-log4j-zero-day.md

```diff
@@ -96,16 +96,20 @@ Almost all versions of log4j version 2 are affected.
 
 `2.0-beta9 <= Apache log4j <= 2.14.1`
 
+:::caution Limited vulnerability in `2.15.0`
+As of Tuesday, Dec 14, version `2.15.0` was found to still have a possible [vulnerability in some apps](https://lists.apache.org/thread/83y7dx5xvn3h5290q1twn16tltolv88f).
+We recommend updating to `2.16.0` which removes the JNDI feature entirely.
```

Comment:
disables (https://github.com/apache/logging-log4j2/commit/44569090f1cf1e92c711fb96dfd18cd7dccc72ea `log4j2.enableJndi=false` default), not removes

Reply:
Nice thank you